handler/src/main/java/io/netty5/handler/ssl/SslHandler.java

```diff
@@ -1790,9 +1789,7 @@ public void handlerAdded0(final ChannelHandlerContext ctx) throws Exception {
             startHandshakeProcessing(active);
             // If we weren't able to include client_hello in the TCP SYN (e.g. no token, disabled at the OS) we have to
             // flush pending data in the outbound buffer later in channelActive().
-            final ChannelOutboundBuffer outboundBuffer;
-            if (fastOpen && ((outboundBuffer = channel.unsafe().outboundBuffer()) == null ||
-                    outboundBuffer.totalPendingWriteBytes() > 0)) {
+            if (fastOpen) {
```

Comment:
@chrisvest I think this is fine as `flush` is a no-op if there is nothing in the outbound buffer.